### PR TITLE
Allow user to add and display task

### LINKED ISSUE #32: Feature: Allow user to add and display task in ToDo list
**Is your feature request related to a problem? Please describe.**
User must be allowed to create tasks in app.

**Describe the solution you'd like**

- Click on "Add Task" button on `Navbar`,
- After that display modal from Angular Material to allow filling up name, description and completion properties of task,
- Modal's buttons: "Add" and "Cancel", first adds task to table, second just closes modal without saving,

**Describe alternatives you've considered**

**Additional context**
End2End

### PR DESCRIPTION
**Pull request by Lukash88**
## Summary

Allow user to add and display a new Task in ToDo list.

## Control List

- [x] Code formatting checked,
- [x] Coding Guidelines compliance checked,
- [ ] Documentation updated (if necessary),
- [ ] Automated tests updated (if necessary),

## Screenshots

image

image

image

### CODE REVIEW

File: binder-web-frontend/src/pages/home/components/navbar/navbar.component.ts [hunk 1]
```diff
@@ -2,8 +2,10 @@ import { Component } from '@angular/core';
 import { MatDialog } from '@angular/material/dialog';
 import { Subject, takeUntil } from 'rxjs';
 import { TableDialogComponent } from 'src/pages/home/components/table-dialog/table-dialog.component';
-import { DefaultTable, TablesService } from 'src/api';
-import { dialogConfig } from 'src/shared/consts/appConsts';
+import { DefaultTable, TablesService, ToDoTask, ToDoTasksService } from 'src/api';
+import { tableDialogConfig, taskDialogConfig } from 'src/shared/consts/appConsts';
+import { TaskDialogComponent } from '../task-dialog/task-dialog.component';
```
Comment (StraykerPL) [inline binder-web-frontend/src/pages/home/components/navbar/navbar.component.ts]: Change to absolute path

File: binder-web-frontend/src/pages/home/components/navbar/navbar.component.ts [hunk 2]
```diff
@@ -53,6 +62,43 @@ export class NavbarComponent {
...
+    const dialogRef = this.dialog.open(TaskDialogComponent, taskDialogConfig);
+
+    dialogRef
+      .afterClosed()
+      .pipe(takeUntil(this.subscribe$))
+      .subscribe({
+        next: (result) => {
+          result.id = this.currentlySelectedTableId;
+          this.newTask = result;   
+          console.log(result);
```
Comment (StraykerPL) [inline binder-web-frontend/src/pages/home/components/navbar/navbar.component.ts]: Remove logging.

File: binder-web-frontend/src/pages/home/components/task-dialog/task-dialog.component.scss
Comment (StraykerPL) [inline binder-web-frontend/src/pages/home/components/task-dialog/task-dialog.component.scss:1]: Too many percentages on width/height values. Is it possible to replace someof them?

**Review by StraykerPL — CHANGES_REQUESTED** [review]

File: binder-web-frontend/src/pages/home/components/task-dialog/task-dialog.component.scss
Comment (StraykerPL) [reply binder-web-frontend/src/pages/home/components/task-dialog/task-dialog.component.scss:1]: Discussed internally.

File: binder-web-frontend/src/pages/home/components/navbar/navbar.component.ts (hunk 1 above)
Comment (Lukash88) [reply binder-web-frontend/src/pages/home/components/navbar/navbar.component.ts]: Resolved 6b16d83.

File: binder-web-frontend/src/pages/home/components/navbar/navbar.component.ts (hunk 2 above)
Comment (Lukash88) [reply binder-web-frontend/src/pages/home/components/navbar/navbar.component.ts]: Resolved 6b16d83.

File: binder-web-frontend/src/pages/home/components/task-dialog/task-dialog.component.scss
Comment (Lukash88) [reply binder-web-frontend/src/pages/home/components/task-dialog/task-dialog.component.scss:1]: Resolved 6b16d83.

**Review by StraykerPL — APPROVED** [review]

**Pull request merged**